src/index.html

```diff
@@ -10,7 +10,29 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="search__form search__form__big" data-qa="big">
+      <form class="search__field">
+        <input type="text"
+          class="search search__big"
+          name="searchBig"
+          placeholder="Try &quot;Los Angeles&quot;"
+      >
+      </form>
+    </div>
+
+    <div class="search__form search__form__small" data-qa="small">
```

Comment:
Wow, looks like BEM 👍 

But in BEM modifiers are separated with '--', '__' is for elements and elements should have a block, but its ok, you will learn about it later 🙂